ospatch/yum_update.go

```diff
@@ -78,6 +78,14 @@ func YumDryRun(dryrun bool) YumUpdateOption {
 	}
 }
 
+func packageNameWithArch(pkg *packages.PkgInfo) string {
+	nameWithArch := pkg.Name
+	if len(pkg.Arch.RawArch) > 0 {
```

Comment:
For the consistency i'm encourage you to create method to expose the data rather than just use field, but i will not insist on that

Reply:
I think usage of mixed model might be misleading, if data model is not straightforward enough to just expose it (make all fields public) we might need to introduce an API (a set of methods that restrict user's usage patterns to the correct one) the problem with current model (i'm not thinking it is blocking for this PR submission, but might be harmful in the future) both fields are still public you're not restricting wrong usage patterns. And one last question that might be a good hint, is there a benefit of not having the method?